crates/rustc_codegen_spirv-types/Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [package]
 name = "rustc_codegen_spirv-types"
 description = "SPIR-V backend types shared between rustc_codegen_spirv and spirv-builder"
+documentation = "https://embarkstudios.github.io/rust-gpu/api/rustc_codegen_spirv_types/index.html"
```

Comment:
Btw, I almost regret naming this crate this, maybe it should be named `spirv-builder-types`...
It's used to communicate between `rustc_codegen_spirv` and `spirv-builder` (through JSON).

You don't need to do this because it should compile on stable:
https://github.com/EmbarkStudios/rust-gpu/blob/733c2d89e1ec99a72729f3b4f4b50b5a474ba93a/crates/rustc_codegen_spirv-types/src/lib.rs#L1-L6